### PR TITLE
OSDOCS#11308: Generating CLI docs for 4.17

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-11308

Link to docs preview:
* https://81546--ocpdocs-pr.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/developer-cli-commands.html
* https://81546--ocpdocs-pr.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/administrator-cli-commands.html

QE review:
- [ ] QE has approved this change.

Additional information:

**Note to peer reviewers: These files are automatically generated. I can't fix anything manually here, but if you see any issues, we can fix it in the appropriate eng repo.**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please QE review?

File: modules/oc-adm-by-example-content.adoc [hunk 1]
```diff
@@ -340,31 +340,72 @@ Create a new project
...
+Monitor new nodes being added to an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Monitor a single node being added to a cluster
+  oc adm node-image monitor --ip-addresses 192.168.111.83
+  
+  # Monitor multiple nodes being added to a cluster by separating each
+  IP address with a comma
```
Comment (bergerhoffer) [inline modules/oc-adm-by-example-content.adoc:378]: Note that I've submitted a fix for this line (should be commented out) here: https://github.com/openshift/oc/pull/1872

File: modules/oc-adm-by-example-content.adoc [hunk 2]
```diff
@@ -340,31 +340,72 @@ Create a new project
...
 
 
+== oc adm node-image create
+Create an ISO image for booting the nodes to be added to the target cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Create the ISO image and download it in the current folder
+  oc adm node-image create
```
Comment (zhouying7780) [inline modules/oc-adm-by-example-content.adoc:350]:
```./oc adm node-image create
error: stat nodes-config.yaml: no such file or directory```
command failed.

File: modules/oc-adm-by-example-content.adoc [hunk 3]
```diff
@@ -340,31 +340,72 @@ Create a new project
...
+  
+  # Use a different assets folder
+  oc adm node-image create --dir=/tmp/assets
```
Comment (zhouying7780) [inline modules/oc-adm-by-example-content.adoc:353]: same here : ```./oc adm node-image create --dir=/tmp/assets
error: stat nodes-config.yaml: no such file or directory```

File: modules/oc-adm-by-example-content.adoc [hunk 4]
```diff
@@ -340,31 +340,72 @@ Create a new project
...
+  
+  # Specify a custom image name
+  oc adm node-image create --o=my-node.iso
```
Comment (zhouying7780) [inline modules/oc-adm-by-example-content.adoc:356]:
```./oc adm node-image create --o=my-node.iso
error: unknown flag: --o
See 'oc adm node-image create --help' for usage```

File: modules/oc-adm-by-example-content.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/oc-adm-by-example-content.adoc:350]: @zhouying7780 if I look at the help for `oc adm node-image create`, it does say this:

> A nodes-config.yaml config file must be created to provide the required initial configuration for the selected nodes.

So I think this means that this file must exist (either in the current folder for this command, or in the /tmp/assets/ folder for the next command) for these commands to work.

File: modules/oc-adm-by-example-content.adoc (hunk 4 above)
Comment (bergerhoffer) [reply modules/oc-adm-by-example-content.adoc:356]: I'll submit a PR to fix this

Comment (bergerhoffer) [reply modules/oc-adm-by-example-content.adoc]: Submitted https://github.com/openshift/oc/pull/1874 to fix this in the oc repo for next time. Also manually fixed it here in this PR

File: modules/oc-adm-by-example-content.adoc (hunk 2 above)
Comment (andfasano) [reply modules/oc-adm-by-example-content.adoc:350]: That's correct. If the `--dir` option is not configured (and no single node flag is used), by default the asset is equal to the current working directory

File: modules/oc-adm-by-example-content.adoc (hunk 3 above)
Comment (andfasano) [reply modules/oc-adm-by-example-content.adoc:353]: In this example the `nodes-config.yaml` it's expected to be in the `/tmp/assets` folder

**Pull request merged**